### PR TITLE
Travis CI: Add those Py3 pytests that already pass

### PR DESCRIPTION
**Pull request by cclauss**
Add `pytest openlibrary/mocks openlibrary/olbase openlibrary/utils scripts/tests` to the tests that __must__ pass on Python 3.  We will add others to this list in future PRs.

Also, we can accelerate Python 3 tests because Node.js is now compatible with Python 3.

Closes #

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,11 +8,14 @@ jobs:
...
     - name: “Python 2.7 on bionic”
       python: "2.7"
-    - name: “Python 3.8 make lint only”
+    - name: “Python 3.8 make lint and some pytests”
       python: "3.8"
       install: pip install flake8
       script:
         - make lint
+        - pip install -r requirements_test.txt
+        - pushd vendor/infogami && git pull origin master && popd
```
Comment (hornc) [inline .travis.yml]: these 2 lines should be in the `install` section.

I was going to investigate whether there is a better way to get the latest `infogami` using `git submodule sync` or similar, but that can probably be a separate PR since the current method uses the `pushd` and `git pull` method.

Comment (cclauss) [reply .travis.yml]: Blocked by https://github.com/internetarchive/openlibrary/pull/2951

**Comment by cclauss** [comment]
`make lint` must happen before `npm install` because of  #2951 so I left a TODO in the comments.

**Comment by mekarpeles** [comment]
This lgtm; merging as @cclauss is blocked

**Comment by hornc** [comment]
@mekarpeles I've just noticed this was merged. I don't understand some of the latest changes, the script and install sections seem more mixed than before, and it looks like this should have been squashed as there are a lot of intermediate changes now in the commit history.   What has this got to do with #2951 and what was blocking ?

**Comment by cclauss** [comment]
Workaround 1. We lint before we install dependencies because of #2951 
Workaround 2. #3124 because of #2951 

Instead of workarounds, why not just fix #2951 and remove the workarounds?

**Comment by hornc** [comment]
2. #3124 wasn't supposed to be a workaround -- we shouldn't be linting and blocking on code we don't maintain --  #3124 was simply making the two exclusions match, as one line excluded `node_modules` but the first one didn't have the full exclusion list.   I was trying to lint Open Library python code locally using my dev docker image, which is what prompted #3124  I missed this PR being merged, but the Node dependencies issue seem completely different to me.

**Comment by cclauss** [comment]
> we shouldn't be linting and blocking on code we don't maintain 

This is where I disagree.  Python is a dynamic language so a syntax error in a dependency will halt our code in front of our users.  If we pull code into our runtime then we should be alerted to and concerned if that code can crash our service.

Looking at: `FLAKE_EXCLUDE=./.*,scripts/20*,vendor/*,node_modules/*`
* `./.*` is only required because we have other excludes...
* `scripts/20*` is dead code containing syntax errors and should be deleted from this repo
* `vendor/*` should all be flake8 tested upstream so retesting at this level is good redundancy
* `node_modules/*` the only failure is because of #2951 which I fixed upstream 10 months ago.  I prefer that we are alerted to syntax errors, etc. that appear in our runtime dependencies.

I believe that we should get rid of the excludes because they create blind spots that prevent us from seeing bugs that can crash our service.
